web-console/future-openshift/dashboards/dashboards.md

```diff
@@ -0,0 +1,34 @@
+# Dashboards
+
+## Dashboard Organization
+
+### Dashboard-related navigation changes
+![Dashboard navigation](img/dashboard-navigation.png)
+- The `Status` navigation item will be removed and replaced with `Dashboards`.
+- If technically feasible, `Dashboards` should be the landing page of OpenShift and should be the first navigation item, followed by `Projects`. If not, the order should remain unchanged.
+- The status page for "All Projects" will be removed and replaced with the new cluster scoped overview dashboard
+- Project specific status pages will become tabs of a detail view for each project. See "Object scoped dashboards" below.
+
+### Cluster scoped dashboards
+![Dashboard navigation](img/cluster-scoped.png)
+
+#### Overview dashboard
+- The `Overview` dashboard is the primary tab on the dashboards page and should provide high-level information about the cluster and its components. Specific notes on the content of this dashboard are found elsewhere.
+
+#### Specialized dashboards
+- Additional dashboard tabs may be added for specific parts of OpenShift or potentially by Operators. These should be placed to the right of the `Overview` dashboard tab and can be organized as necessary for the relevant information
+
+### Object scoped dashboards
+- The `Overview` tab on object details pages should primarily be used to illustrate the YAML of the object in a visual way. If applicable, a few select charts with important metrics may be included on the page.
```

Comment:
Letting `Overview` be its usual YAML-exposing self sounds good for now. As we keep exploring this, I do kind of wonder if a separate `Dashboard` tab is necessary at all. Maybe all of those dashboard-ey cards could be placed below a mandatory, top-aligned, fullwidth `Details` card of an `Overview` tab. We'll figure that out as we go. 🙂